README.org

```diff
@@ -1,17 +1,33 @@
+* Eventually Matrix
+
+  [[https://en.wikipedia.org/wiki/Eventual_consistency][Eventually]] Matrix is a
+  repository of knowledge about the Matrix ecosystem maintained by the community
+  for the community. It is the "bleeding edge" counterpart of the official
+  Matrix.org repositories in terms of documentation, SDK and libraries.
+
 * Purpose                                              :matrix:eventually_matrix:
 
-  Eventually Matrix is a repository of knowledge about the Matrix ecosystem maintained
-  by the community for the community. In broad strokes, this repository exists for
-  the following purposes in roughly the order they're listed.
+  In broad strokes, this repository exists for the following purposes in roughly the
+  order they're listed.
 
+  - Allow Matrix community to easily document the current behavior of implementations
+    in a specification format when such behaviour deviates from the upstream spec.
+    - Speed up the process of adoption and validation of new features/endpoints.
+    - Give a more bleeding-edge reference document that Matrix community can use to
```

Comment:
Same comments as on "bleeding edge" above. Would just omit this bullet point entirely, imo.

Reply:
I'm thinking:
> Provide the latest available knowledge that the Matrix community can use to prepare for stable releases of the official specification.